docs/core/allocation.qmd

```diff
@@ -94,25 +94,39 @@ There are 2 types of variables whose value has to be determined to solve the all
 $$
     A^\text{basin}_{\hat{i}} \ge 0, \quad \hat{B}_S,
 $$
-where $\hat{B} = m_S(B_S) \subset \hat{V}_S$ is the set of basin node ids in the allocation graph.
+where $\hat{B}_S = m_S(B_S) \subset \hat{V}_S$ is the set of basin node ids in the allocation graph.
 
 :::{.callout-note}
 Currently the basin allocations are not taken into account in the implementation.
 :::
 
 ### The optimization objective
 
-The goal of allocation is to maximize the flow to the users. Therefore, we use the following optimization objective:
+The goal of allocation is to get the flow to the users as close as possible to their demand. To achieve this, the following objectives are supported:
```

Comment:
Besides the equations, could you add a bit more text here on the differences? Like that quadratic focuses more on avoiding large shortages, and that for linear it is all the same.